crates/oxc_language_server/src/linter/server_linter.rs

```diff
@@ -24,9 +24,73 @@ pub struct ServerLinter {
 }
 
 impl ServerLinter {
+    pub fn new(root_uri: &Uri, options: &Options) -> Self {
+        let nested_configs = Self::create_nested_configs(root_uri, options);
+        let root_path = root_uri.to_file_path().unwrap();
+        let relative_config_path = options.config_path.clone();
+        let oxlintrc = if relative_config_path.is_some() {
+            let config = root_path.join(relative_config_path.unwrap());
+            if config.try_exists().expect("Could not get fs metadata for config") {
```

Comment:
should thie be `.ok()` rather than panicing?

Reply:
`ok()` is not enough, changed it to `if config.try_exists().is_ok_and(|exists| exists) {`

Reply:
ah yes you're right